api-server/api/controllers/api_impl.ts

```diff
@@ -0,0 +1,222 @@
+import * as express from 'express';
+import {
+  ArtemisJolokia,
+  JolokiaExecResponse,
+  JolokiaObjectDetailsType,
+  JolokiaReadResponse,
+} from '../apiutil/artemis_jolokia';
+import { API_SUMMARY } from '../../utils/server';
+
+// get the broker mbean from jolokia
+export function getBrokers(_: express.Request, res: express.Response): void {
```

Comment:
Could you please update the code to follow the ES6 syntax, as we follow in the project's codebase. (i.e)

```
export const getQueues = (req: express.Request, res: express.Response): void => {
   // rest of your code here
}
```

Reply:
ok will do. Thanks!